conf/application.conf

```diff
@@ -163,47 +163,44 @@ versioning {
 
 api-config {
   scopes {
-    "dummy-scope" {
-      fields = [ "A","B","C","D","E","F","G","H","I","J","K","L" ]
+    "read:individuals-details-laa-c3" {
+       fields = ["A", "B", "C", "D", "E", "F", "I", "J"]
     }
-    "read:hello-scopes-1" {
-      fields = [ "A","B","C"]
+    "read:individuals-details-laa-c4" {
+       fields = ["A", "B", "C", "D", "E", "F", "I", "J", "G", "H"]
     }
-    "read:hello-scopes-2" {
-      fields = [ "A","B"]
+    "read:individuals-details-hmcts-c3" {
+       fields = ["A", "B", "C", "D", "E", "F", "I", "J"]
+    }
+    "read:individuals-details-hmcts-c4" {
+       fields = ["A", "B", "C", "D", "E", "F", "I", "J", "G", "H"]
+    }
+    "read:individuals-details-lsani-c1" {
+       fields = ["A", "B", "C", "D", "E", "F", "I", "J"]
+    }
+    "read:individuals-details-lsani-c3" {
+       fields = ["A", "B", "C", "D", "E", "F", "I", "J"]
+    }
+    "read:individuals-details-nictsejo-c4" {
+       fields = ["A", "B", "C", "D", "E", "F", "I", "J"]
     }
   }
 
   endpoints {
-    paye {
-      endpoint = "/individuals/employments/paye?matchId=$randomMatchId{&fromDate,toDate}"
-      fields {
-        A = "payments"
-        B = "employer/employerName"
-        C = "employer/employerAddress/line1"
-        D = "employer/employerAddress/line2"
-        E = "employer/employerAddress/line3"
-        F = "employer/employerDistrictNumber"
-        G = "employer/employerSchemeReference"
-        H = "employmentStartDate"
-        I = "employmentLeavingDate"
-        J = "employmentPayFrequency"
-        K = "employeeAddress"
-        L = "payrollId"
+    details {
+        endpoint = "/individuals/details/"
```

Comment:
I don't think this needs to be a route. It just needs to match the endpoint name... 

I.e. https://github.com/hmrc/individuals-details-api/blob/master/app/uk/gov/hmrc/individualsdetailsapi/controllers/ContactDetailsController.scala#L35